win32/src/socket_transport_win32.c

```diff
@@ -628,7 +628,7 @@ SOCKET_ACCEPT_RESULT socket_transport_accept(SOCKET_TRANSPORT_HANDLE socket_tran
 
                 read_fds.fd_array[0] = socket_transport->socket;
                 read_fds.fd_count = 1;
-                timeout.tv_usec = 1000 * 100;
+                timeout.tv_usec = connection_timeout_ms * 1000;
```

Comment:
>1000 [](http://example.com/codeflow?start=58&length=4)

You probably create a define at the top of the file for this, since you have it in 2 places